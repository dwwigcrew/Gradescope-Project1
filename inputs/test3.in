3
3
107
10
212