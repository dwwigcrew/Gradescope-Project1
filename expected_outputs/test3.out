
Welcome to the ENEE140 pseudo-random number generator!
1: Print RND_MAX
2: Generate uniformly-distributed positive integers
3: Generate uniformly-distributed positive integers, up to a given limit
4: Generate uniformly-distributed integers, from a given range
5: Generate uniformly-distributed floats, from a given range
6: Generate exponentially-distributed floats

Enter your choice (1-6): 
Select the algorithm to use.

Enter your choice (1-3): 
Select the seed for the random number generator: 
How many numbers should I generate: 
Enter the maximum number to generate: 208 75 151 137 16 131 49 77 153 29 
